src/components/home/style.scss

```diff
@@ -0,0 +1,59 @@
+.footerNav {
+  height: 100px;
+  background-color: #05225a;
+  display: flex;
+  justify-content: space-around;
+  color: white;
+}
+
+.footerNav {
+  list-style: none;
+}
+footer {
```

Comment:
newline above. give `footer` a class, its not good to use html tags because it affects all `footer` on the page in case someone else uses that again for some reason.